### PR TITLE
chore: Change module version.

### PR DESCRIPTION
**Pull request by u5surf**
After merging https://github.com/vozlt/nginx-module-vts/pull/240

I found a lack of changing module version...

@SuperQ 
If it does not  automate here, it should be change every time. What can you do that?

### CODE REVIEW

**Comment by SuperQ** [comment]
I guess we should add that to the Release instructions in the README.

Something like this:
```
sed -i "s/NGX_HTTP_VTS_MODULE_VERSION \".*/NGX_HTTP_VTS_MODULE_VERSION \"${version}\"" src/ngx_http_vhost_traffic_status_module.h
git add src/ngx_http_vhost_traffic_status_module.h
```

**Comment by u5surf** [comment]
@SuperQ I added that. Thanks.

File: README.md [hunk 1]
```diff
@@ -1821,6 +1821,8 @@ To cut a release, create a changelog entry PR with [git-chglog](https://github.c
     git-chglog -o CHANGELOG.md --next-tag "${version}"
     git add CHANGELOG.md
     git-chglog -t .chglog/RELNOTES.tmpl --next-tag "${version}" "${version}" | git commit -F-
+    sed -i "s/NGX_HTTP_VTS_MODULE_VERSION \".*/NGX_HTTP_VTS_MODULE_VERSION \"${version}\"" src/ngx_http_vhost_traffic_status_module.h
+    git add src/ngx_http_vhost_traffic_status_module.h
```
Comment (SuperQ) [inline README.md:1825]: This should go before the commit.

File: src/ngx_http_vhost_traffic_status_module.h [hunk 2]
```diff
@@ -16,7 +16,7 @@
 #include "ngx_http_vhost_traffic_status_string.h"
 #include "ngx_http_vhost_traffic_status_node.h"
 
-#define NGX_HTTP_VTS_MODULE_VERSION  "0.1.19.dev.91bdb14"
+#define NGX_HTTP_VTS_MODULE_VERSION  "0.2.0.dev.b740132"
```
Comment (SuperQ) [inline src/ngx_http_vhost_traffic_status_module.h:19]: I wonder if we should make this the current release. Maintaining it as a hash is a bit circular.

```suggestion
#define NGX_HTTP_VTS_MODULE_VERSION  "v0.2.0"
```

**Review by vozlt — COMMENTED** [review]
https://github.com/vozlt/nginx-module-vts/pull/241#discussion_r963020416

File: README.md (hunk 1 above)
Comment (u5surf) [reply README.md:1824]: @SuperQ Sorry, I fixed it.

**Review by SuperQ — APPROVED** [review]

File: src/ngx_http_vhost_traffic_status_module.h (hunk 2 above)
Comment (SuperQ) [reply src/ngx_http_vhost_traffic_status_module.h]: Seems like a good plan. Let's document this idea as comment on the `#define`

File: src/ngx_http_vhost_traffic_status_module.h [hunk 3]
```diff
@@ -16,7 +16,7 @@
 #include "ngx_http_vhost_traffic_status_string.h"
 #include "ngx_http_vhost_traffic_status_node.h"
 
-#define NGX_HTTP_VTS_MODULE_VERSION  "0.1.19.dev.91bdb14"
+#define NGX_HTTP_VTS_MODULE_VERSION  "v0.2.0"
```
Comment (SuperQ) [inline src/ngx_http_vhost_traffic_status_module.h:19]:
```suggestion
/*
 * This version should follow the stable releases.
 * The format should follow https://semver.org/
 * 
 * If a change has some important impact, include the commit short hash here.
 * I.E "v0.2.0+h0a1s2h"
 *
 */
#define NGX_HTTP_VTS_MODULE_VERSION  "v0.2.0"
```

**Review by vozlt — APPROVED** [review]

**Pull request merged**